tests/zexperimental/test_mcmc.py

```diff
@@ -156,6 +156,128 @@ def test_MALA_regression_warmup(target: cuqi.density.Density):
     sampler_new = cuqi.experimental.mcmc.MALANew(target, scale=1)
     assert_true_if_warmup_is_equivalent(sampler_old, sampler_new)
 
+# ============ LinearRTO ============
+
+@pytest.mark.parametrize("target", targets)
```

Comment:
I was thinking you could test on targets+multiple_likelihood_posterior_targets here. Then remove the redundant lines below

Reply:
Done as suggested; I defined a `LinearRTO_targets` as:
```python
LinearRTO_targets = targets + [
    create_MultipleLikelihoodPosterior_target(dim=32),
    create_MultipleLikelihoodPosterior_target(dim=64),
    create_MultipleLikelihoodPosterior_target(dim=128)
]
```